### ISSUE TITLE
Install fails trying to copy .gitignore

### ISSUE THREAD

**Issue by rjmackay** [yahoo/generator-fluxible#37]
I'm seeing the following error during install.. looking at whats been installed .gitignore is in fact missing. Maybe it went missing when this was pushed to npm?

```
events.js:85
      throw er; // Unhandled 'error' event
            ^
AssertionError: Trying to copy from a source that does not exist: /Users/robbie/.nvm/versions/node/v0.12.0/lib/node_modules/generator-fluxible/app/templates/.gitignore
    at EditionInterface.exports._copySingle (/Users/robbie/.nvm/versions/node/v0.12.0/lib/node_modules/generator-fluxible/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/Users/robbie/.nvm/versions/node/v0.12.0/lib/node_modules/generator-fluxible/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:23:17)
    at EditionInterface.module.exports [as copyTpl] (/Users/robbie/.nvm/versions/node/v0.12.0/lib/node_modules/generator-fluxible/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy-tpl.js:16:8)
    at template (/Users/robbie/.nvm/versions/node/v0.12.0/lib/node_modules/generator-fluxible/node_modules/yeoman-generator/lib/actions/actions.js:134:11)
    at module.exports.yeoman.generators.Base.extend.writing.config (/Users/robbie/.nvm/versions/node/v0.12.0/lib/node_modules/generator-fluxible/app/index.js:46:18)
    at /Users/robbie/.nvm/versions/node/v0.12.0/lib/node_modules/generator-fluxible/node_modules/yeoman-generator/lib/base.js:428:16
    at processImmediate [as _immediateCallback] (timers.js:358:17)
```

**Comment by redonkulus** [comment]
Which version of `yo` do you have installed? Is it the latest?

**Comment by rjmackay** [comment]
Looks like this was broken by v1.1.0 .. 1.0.8 still works

**Comment by rjmackay** [comment]
Yo version is 1.4.7

**Comment by redonkulus** [comment]
Ya it fails locally for me too, @mridgway is fixing.

**Comment by mridgway** [comment]
Sorry about that. Npm apparently renamed one of the template files causing it to not be found. Should be fixed in v1.1.1.

**Comment by rjmackay** [comment]
Great thanks for the amazing response time!
